sunpy/net/scraper.py

```diff
@@ -80,28 +77,31 @@ class Scraper:
     pattern looks with the actual time.
     """
 
-    def __init__(self, pattern, regex=False, **kwargs):
-        if regex:
-            self.pattern = pattern
-            if kwargs:
-                warn_user('regexp being used, the extra arguments passed are being ignored')
-        else:
-            self.pattern = pattern.format(**kwargs)
+    def __init__(self, pattern, **kwargs):
+        pattern = pattern.format(**kwargs)
+        timepattern = pattern
+        for k, v in TIME_CONVERSIONS.items():
+            if k in timepattern:
+                timepattern = timepattern.replace(k,v)
+        self.timepattern = timepattern
+        if "year:4d" in pattern and "year:2d" in pattern:
+            pattern = pattern.replace("year:2d", ":2d")
```

Comment:
Does this happen?

Reply:
yes, for example [here](https://github.com/sunpy/sunpy/blob/813592baab6181288d8d50d6676d585374554960/sunpy/net/tests/test_scraper.py#L255).